config/common.php

```diff
@@ -2,11 +2,17 @@
 
 declare(strict_types=1);
 
-/**
- * @var $params array
- */
-
 use Composer\InstalledVersions;
+use HttpSoft\Message\RequestFactory;
```

Comment:
1. its in `required-dev` section
2. no need to configure your own http bridge, let user decide which package he want to use. 
so just remove these lines, they are not used anyway